src/main/java/creativei/dao/InquiryDao.java

```diff
@@ -11,10 +13,21 @@
 import org.springframework.data.repository.query.Param;
 
 import javax.transaction.Transactional;
+import java.util.Date;
 import java.util.List;
 
 @Transactional
-public interface InquiryDao extends JpaRepository<Inquiry,Long>,JpaSpecificationExecutor<Inquiry>{
-    List<Inquiry> findByInquiryStatus(@Param("status")InquiryStatus status);
-    List<Inquiry> findByInquiryStatusAndIsAttended(InquiryStatus inquiryStatus,Boolean isAttended);
+public interface InquiryDao extends JpaRepository<Inquiry, Long>, JpaSpecificationExecutor<Inquiry> {
+    List<Inquiry> findByInquiryStatus(@Param("status") InquiryStatus status);
+
+    List<Inquiry> findByInquiryStatusAndIsAttended(InquiryStatus inquiryStatus, Boolean isAttended);
+
+    @Query("select count(inq.id) from Inquiry inq where inq.inquiryDate between :inquiryDateTo And :inquiryDateFrom")
+    Integer findCountByInquiryDate(@Param("inquiryDateTo") Date inquiryDateTo, @Param("inquiryDateFrom") Date inquiryDateFrom);
+
+    @Query("select count(inq.id) from Inquiry inq join inq.followUps follUp  where inq.inquiryDate between :inquiryDateTo And :inquiryDateFrom and follUp.caseIndex=:caseIndex3 or follUp.caseIndex=:caseIndex4")
```

Comment:
Rename param to inquiryToDate and inquiryFromDate

Reply:
name changed e224e82